packages/blueprints-integration/src/context.ts

```diff
@@ -148,7 +148,15 @@ export interface ISegmentUserContext extends IUserNotesContext, IRundownContext,
 }
 
 /** Actions */
-export interface IActionExecutionContext extends IShowStyleUserContext, IEventContext {
+export interface IDataStoreActionExecutionContext extends IShowStyleUserContext, IEventContext {
+	setTimelineDatastoreValue(key: string, value: any, mode: 'temporary' | 'indefinite'): Promise<void>
+	removeTimelineDatastoreValue(key: string): Promise<void>
+}
+
+export interface IActionExecutionContext
+	extends IShowStyleUserContext,
```

Comment:
Should this be extending `IDataStoreActionExecutionContext`? Is there any value in letting blueprints set datastore values during the 'slow' method?

Reply:
Well, if you need some of the data from the cache in order to make a decision you will have to use the 'slow' method. Maintenance wise it's cheap enough to leave it in, so I don't want to make the decision to limit the usage in that way right now.